src/basic/format-util.h

```diff
@@ -74,8 +74,9 @@ typedef enum {
 
 #define FORMAT_BYTES_MAX 16U
 
-char *format_bytes_full(char *buf, size_t l, uint64_t t, FormatBytesFlag flag);
+char *format_bytes_full(char *buf, size_t l, uint64_t t, FormatBytesFlag flag) _warn_unused_result_;
 
+_warn_unused_result_
```

Comment:
Why warn if the return value is unused? Isn't this function simply returning the `buf` parameter for convenience reasons?

Reply:
Most of the time, this function is used through the macro. And when the macro is used, it is certainly an error. We cannot set the attribute just on the macro, so it is set on the function.